frontend/coprs_frontend/coprs/logic/modules_logic.py

```diff
@@ -157,16 +157,18 @@ def platform_chroots(self):
         """
 
         # Just to be sure, that all chroot abbreviations from platform are in expected format, e.g. f28 or -f30
-        for abbrev in self.platform:
-            if not (abbrev.startswith(("f", "-f")) and abbrev.lstrip("-f").isnumeric()):
-                raise ValidationError("Unexpected platform '{}', it should be e.g. f28 or -f30".format(abbrev))
+        self._validate_platform()
 
         chroot_archs = {}
         for chroot in self.copr.active_chroots:
             chroot_archs.setdefault(chroot.name_release, []).append(chroot.arch)
 
         def abbrev2chroots(abbrev):
-            name_release = abbrev.replace("-", "").replace("f", "fedora-")
+            abbrev = abbrev.replace("-", "")
+            if abbrev.startswith("f"):
+                name_release = abbrev.replace("f", "fedora-")
+            elif abbrev.startswith("el"):
+                name_release = abbrev.replace("el", "epel-")
```

Comment:
Typically I translate "el" into "rhel" (or just Enterprise Linux).  Can you please document why epel?  Would this work against the RHEL chroots?

Reply:
I updated the PR, to match "el" to both EPEL and RHEL. This way whatever of those chroots will be enabled in the project, the module will be built for it.